modules/reitit-frontend/src/reitit/frontend/history.cljs

```diff
@@ -59,6 +59,14 @@
           el
           (recur (.-parentNode el)))))))
 
+(defn- event-target [event]
+  "Read event's target from composed path to get shadow dom working,
+   fallback to target property if not available"
+  (let [original-event (.-event_ event)]
```

Comment:
Use `.getBrowserEvent` method instead of the internal `event_` property.

Reply:
Changed